### ISSUE TITLE
AKO should support for selector-less load balancer type of service.

### ISSUE THREAD

**Issue by XudongLiuHarold** [vmware/load-balancer-and-ingress-services-for-kubernetes#313]
AKO should add support for selector-less `LoadBalacaner` type of service for customizing endpoints.

AKO needs to use IPs and Ports directly from `Endpoints` object to create Pool Members for a `selector-less` LoadBalancer type of Service. 

For example, if I want to specify endpoints by my own like this:
```yaml
apiVersion: v1
kind: Service
metadata:
 name: HA-lb
spec:
  ports:
  - protocol: TCP
    port: 6443
    targetPort: 6443
  type: LoadBalancer
---
apiVersion: v1
kind: Endpoints
metadata:
  name: ha-lb-eps
  namespace: default
subsets:
- addresses:
  - ip: 100.96.0.2
    nodeName: controlplane-node-1
  - ip: 100.96.0.7
    nodeName: controlplane-node-2
  ports:
  - port: 6443
    protocol: TCP
```

AKO should directly add IPs and Ports from `Endpoints` `ha-lb-eps` to virtual service's pool members.

**Comment by sudswasavi** [comment]
@XudongLiuHarold how about your create the ep with the same name as the `service`?
That should solve your use case I think.

**Comment by XudongLiuHarold** [comment]
> @XudongLiuHarold how about your create the ep with the same name as the `service`?
> That should solve your use case I think.

Hi @sudswasavi , I updated the description, sorry for the misleading.

**Comment by andrewsykim** [comment]
@XudongLiuHarold I think the issue description is still misleading. I believe selectorless Services are supported already, but I think you're asking for the serviceType (ClusterIP or NodePorT) to be configurable per Service LoadBalancer right?

**Comment by sudswasavi** [comment]
This should be available in AKO 1.4.1
